src/rust/engine/options/src/flags_tests.rs

```diff
@@ -54,21 +65,20 @@ fn test_string() {
 
     assert_string("bar", option_id!("foo"));
     assert_string("cheese", option_id!("baz", "spam"));
-    assert_string("swallow", option_id!(-'u', "unladen", "capacity"));
+    assert_string("swallow", option_id!("unladen", "capacity"));
     assert_string("debug", option_id!(-'l', "level"));
     assert_string("qux", option_id!(["scope"], "qux"));
     assert_string("quux", option_id!(["scope"], "quux"));
 
     assert!(args.get_string(&option_id!("dne")).unwrap().is_none());
-    assert!(args.get_passthrough_args().is_none());
 }
 
 #[test]
 fn test_bool() {
     let args = mk_args([
-        "-c=swallow",
+        "--unladen-capacity=swallow",
```

Comment:
The old `Args` implementation happened to support arbitrary short flags, and we tested that, but the OptionsParser itself didn't support those, so this is not taking away any capability of Pants itself, just an overeagerness of the previous implementation.